src/components/ProductItem/ProductItem.tsx

```diff
@@ -158,15 +162,15 @@ export const ProductItem: FunctionComponent<ProductProps> = ({
     ? setRoute({ sku: productView?.sku, urlKey: productView?.urlKey })
     : product?.canonical_url;
 
-  const handleAddToCart = async () => {
+  const updateCart = async (selectedVariants: string[] = []) => {
     setError(false);
     if (isSimple) {
```

Comment:
will make changes here.
I think we don't need this check since "Quick Add" button will be only available for sportswear category

Reply:
removed isSimple check